### PR TITLE
Extend widget props to WrapperWidgetUI

### PR DESCRIPTION
**Pull request by juandjara**
with a better implementation than in #50

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # CHANGELOG
 
 ## Not released
+
 - Pass actions and options to WrapperWidgetUI from HistogramWidget and CategoryWidget [#50](https://github.com/CartoDB/carto-react-lib/pull/50)
-- Fix addSource in cartoSlice spreading layerAttributes [#52∫](https://github.com/CartoDB/carto-react-lib/pull/52)
+- Fix addSource in cartoSlice spreading layerAttributes [#52](https://github.com/CartoDB/carto-react-lib/pull/52)
+- extend widget props to WrapperWifgetUI with a better implementation than in [#50](https://github.com/CartoDB/carto-react-lib/pull/50) [#57](https://github.com/CartoDB/carto-react-lib/pull/57)
```
Comment (VictorVelarde) [inline CHANGELOG.md:7]:
```suggestion
- Extend widget props to WrapperWidgetUI with a better implementation [#57](https://github.com/CartoDB/carto-react-lib/pull/57)
```

🔝 a typo in Widget and a suggestion to be shorter

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 
 ## 1.0.0-beta8 (2020-12-15)
+
```
Comment (VictorVelarde) [inline CHANGELOG.md:10]: We didn't apply here a linter on markdown, so maybe remove the blanks, don't you think?

File: src/widgets/CategoryWidget.js [hunk 3]
```diff
@@ -19,6 +19,7 @@ import { AggregationTypes } from './AggregationTypes';
   * @param  {formatterCallback} [props.formatter] - Function to format each value returned.
   * @param  {boolean} [props.viewportFilter=false] - Defines whether filter by the viewport or not. 
   * @param  {errorCallback} [props.onError] - Function to handle error messages from the widget.
+  * @param  {Object} [props.wrapperProps] - Extra props to pass to WrapperWidgetUI (see storybook for reference)
```
Comment (neokore) [inline src/widgets/CategoryWidget.js:22]: Maybe this information is not clear enough. Please try to improve it putting a link or explaining that there you can put the props defined for the WrapperWidgetUI in this component documentation.

File: CHANGELOG.md (hunk 1 above)
Comment (juandjara) [reply CHANGELOG.md:7]: Let's go 🚀

File: CHANGELOG.md (hunk 2 above)
Comment (juandjara) [reply CHANGELOG.md:10]: This blanks were added by my editor (VS Code) automatically with Prettier "format on save" plugin, but I can remove them if you want

File: src/widgets/CategoryWidget.js (hunk 3 above)
Comment (juandjara) [reply src/widgets/CategoryWidget.js:22]: Can you put a markdown link like [WrapperWidgetUI](https://storybook-react.carto.com/?path=/docs/widgets-wrapperwidgetui--default) in there?

File: CHANGELOG.md (hunk 2 above)
Comment (VictorVelarde) [reply CHANGELOG.md:12]: Ok, let them in. We'll try to add prettier to the project, as we had before on carto-react-template

File: src/widgets/CategoryWidget.js (hunk 3 above)
Comment (juandjara) [reply src/widgets/CategoryWidget.js]: @neokore I added the link to the storybook in the JSDoc using markdown syntax

**Comment by juandjara** [comment]
@VictorVelarde @neokore can you take a look at this, pls?

**Comment by VictorVelarde** [comment]
LGTM @juandjara, I'm gonna merge this one

**Review by VictorVelarde — APPROVED** [review]

**Pull request merged**